src/tests/unit/doughs-dao.test.js

```diff
@@ -7,6 +7,8 @@ import sinonChai from 'sinon-chai';
 
 import { getDoughsData } from './test-data';
 
+sinon.restore();
```

Comment:
Yeah, I was noticing that `sinon.replace(config, 'get', () => ({ oracledb: {} }));` was causing an error because it was used in multiple modules

Reply:
I think what's happening here is that when the tests are about to be run, they are all imported at the same time. Meaning these top-level replaces/restores are all occurring at once. This isn't ideal since tests are affecting each other. Could you fix this?

Reply:
Looks like moving `sinon.replace` to the top level `beforeEach` call fixes it, unsurprisingly. I do believe, for what it's worth, that Mocha executes tests in serial which is why this didn't cause any errors as is.

Reply:
Right, the tests are executed serially, but all of the modules are required/imported at once. So when one of the modules had already called replace, if another module called replace without first calling restore, there would be an error since the the replacement had already occurred. Also, the config replacement was only lasting until the first call to restore (in `afterEach`) which isn't good since we always want this to be stubbed out. Moving them to a beforeEach is the best solution, since the stub will exist for all of the tests without causing any race conditions.

Reply:
Fair. I can see a lot of ways in which this is the best solution.